backend/src/main/java/org/sefglobal/core/repository/AmbassadorRepository.java

```diff
@@ -10,4 +12,5 @@
 public interface AmbassadorRepository extends JpaRepository<Ambassador, Long> {
     Ambassador findByIdAndStatus(long id, String status);
     List<Ambassador> findAllByStatus(String status);
+    Page<Ambassador> findAll(Pageable pageable);
```

Comment:
You don't need to implement this method as `PagingAndSortingRepository ` already provides one